cloudevents/extensions/bam.md

```diff
@@ -0,0 +1,117 @@
+
+### businesstransactionid
+
+- Type: `String`
+- Description: A unique identifier for the transaction.
+- Constraints
+  - REQUIRED
+  - MUST be a non-empty string
+  - RECOMMENDED as monotonically increasing and contiguous identifier
+    that is lexicographically-sortable.
+
+### businessprocessid
+
+- Type: `String`
+- Description: A unique identifier for the business process
+  associated with the event.
+- Constraints
+  - REQUIRED
+  - MUST be a non-empty string
+  - RECOMMENDED a alphanumeric string that contains non-whitespace characters
+    and only hyphens, underscores, and periods.
+
+### businessprocesstransactionid
+
+- Type: `String`
+- Description: A unique identifier for the business process transaction
+  associated with the event.
+- Constraints
+  - REQUIRED
+  - MUST be a non-empty string
+  - RECOMMENDED a alphanumeric string that contains non-whitespace characters
+    and only hyphens, underscores, and periods.
+
+### businessprocesstransactionstepid
+
+- Type: `String`
+- Description: A unique identifier for the business process transaction step
+  associated with the event.
+- Constraints
+  - REQUIRED
+  - MUST be a non-empty string
+  - RECOMMENDED a alphanumeric string that contains non-whitespace characters
+    and only hyphens, underscores, and periods.
+
+### businessprocesstransactionstepstatus
+
+- Type: `String`
+- Description: The status of the business process transaction step
+  associated with the event.
+- Constraints
+  - OPTIONAL
+  - if present, MUST be a non-empty string
+  - RECOMMENDED a alphanumeric string that contains non-whitespace characters
+    and only hyphens, underscores, and periods.
+
+### businessprocesstransactioncompletion
+
+- Type: `Boolean`
+- Description: Indicates if the business process transaction associated
+  with the event has completed.
+- Constraints
+  - OPTIONAL
+  - if present, MUST be a boolean value
+
+## Usage
+
+When this extension is used, producers MUST set the value of
+the `businesstransactionid`, `businessprocessid`,
+`businessprocesstransactionid`, and `businessprocesstransactionstepid`
+attributes to the unique identifiers of the business process, transaction,
+and transaction step associated with the event.
+
+Intermediaries MUST NOT change the value of the `businesstransactionid`,
+`businessprocessid`, `businessprocesstransactionid`,
+and `businessprocesstransactionstepid` attributes.
+
+## Use cases
+
+This extension can be used in cases in which a business activity monitoring
+system is used to monitor the progress of a business process, and the events
+generated by the process are used to track the progress of the process.
+Usually these systems have their own modelling language to describe the
+business process, and the events generated by the process
+are used to track the progress of the process.
+
+## References
+
+- [Gartner Business Activity Monitoring](https://www.gartner.com/en/information-technology/glossary/business-activity-monitoring-bam)
```

Comment:
I get a 404 for this

Reply:
:) it so happend that they changed their content structure. They started to include the language in their urls.